blackbox/docs/upgrading.txt

```diff
@@ -0,0 +1,122 @@
+.. highlight:: sh
+
+=========
+Upgrading
+=========
+
+Normally, the upgrade process for CrateDB is as simple as installing the new version of the software on each node. Sometimes, however, there may be additional steps you need to take. Please see the corresponding section on this page for instructions.
+
+Some versions CrateDB may introduce backwards incompatible changes.
+
+When upgrading to a version of CrateDB with backwards incompatible changes, please see the corresponding section on this page for instructions.
+
+Upgrading to 1.1
+================
+
+CrateDB 1.1 introduces a new internal table format that is backwards
+incompatible with previous table formats. Future versions of CrateDB will only
+support this new table format. CrateDB 1.1 will be the only release that can
+read all older table formats *and* the new table format.
+
+CrateDB 1.1 includes new cluster checks that will tell you which tables need
+migrating or reindexing if you plan to use future releases of CrateDB. Once
+this has been done, your tables will be forward compatible with future releases.
+
+To get started, upgrade your cluster like normal. When CrateDB 1.1 is running,
+you may now see that your cluster status is yellow rather than green.
+
+.. TODO: screenshot
+
+Depending on your situation, you may then be told that you should either
+migrate or reindex some of your tables.
+
+.. TODO: screenshot
+
+.. TODO: screenshot
+
+If your cluster status is green, skip to the section `Create New Snapshots`_.
+Otherwise, continue to the section `Preparation`_.
+
+Preparation
+------------
+
+Before you continue, use the :ref:`copy_to` command to create a one-off backup
+of each table. You can do this on any node in your cluster.
```

Comment:
we should phrase it that backing up is for your safety and not required by the migration process.
user can backup via snapshots or copy_to, it's up to the user.